lib/dom-model/DOMDecorators.js

```diff
@@ -20,6 +25,41 @@ function makeDecorator(callback) {
     }
 }
 
+function makeDOMNode(target, selector) {
+    const dataNode = new DOMNode();
+    dataNode.node = target;
+    dataNode.selector = selector;
+    target._reactComponentDataNode = dataNode;
+    return dataNode;
+}
+
+function findCommentNode(element, selector) {
+    for(let i = 0; i < element.childNodes.length; i++) {
+        let node = element.childNodes[i];
+        if( node.nodeType === 8
```

Comment:
Theoretically we can use the constat Node.COMMENT_NODE to make the code clearer. Can you check browser support for that? At the very least you could add a comment to make it clear what 8 is